lib/permissions/src/lib.rs

```diff
@@ -206,6 +209,10 @@ impl PermissionBuilder {
         self.object(ObjectType::Workspace, id)
     }
 
+    pub fn system_object(self) -> Self {
+        self.object(ObjectType::System, "system")
```

Comment:
Do we ever want different permission sets for different systems? Maybe this should be `"sdf"`